schema.json

```diff
@@ -57,22 +57,26 @@
 			"default": "60s"
 		},
 		"dynamic_allocator": {
-			"title": "Dynamic workers allocator",
-			"description": "Configuration options to setup auto workers allocator.",
+			"title": "Auto-scaling/dynamic Workers",
+			"description": "Configuration options for auto-scaling/dynamic workers. If not defined, worker auto-scaling is disabled.",
 			"type": "object",
+			"additionalProperties": false,
 			"properties": {
 				"max_workers": {
-					"description": "Maximum number of workers which is allowed to allocate (no more than 100).",
+					"description": "The maximum number of workers that may be dynamically allocated.",
 					"type": "integer",
-					"default": "10"
+					"minimum": 1,
+					"maximum": 100,
+					"default": 10
 				},
 				"spawn_rate": {
-					"description": "The number of workers that can be spawned per NoFreeWorkers error (but up to `max_workers`).",
+					"description": "The maximum number of workers that may be spawned per NoFreeWorkers error (limited to `max_workers`).",
 					"type": "integer",
-					"default": "5"
+					"minimum": 1,
+					"default": 5
```

Comment:
Only question is if there is a different `maximum` here? I realize 100 is the practical maximum, but I was unsure if it was lower for spawn rate.

Reply:
100 (max_workers) is the overall maximum for dynamically allocated workers. `spawn_rate` could be like 1000, but only 100 would be allocated. If `spawn_rate` is 10 for example, then only 10 workers per NoFreeWorkers error would be allocated.

Reply:
So the practical max is just 100.

Reply:
Yeah